packages/angular/projects/cds-angular/accordion/cds-accordion-panel.directive.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, ElementRef, Input, Output, EventEmitter } from '@angular/core';
+import { CdsAccordionPanel } from '@cds/core/accordion';
+
+@Directive({ selector: 'cds-accordion-panel' })
+export class CdsAccordionPanelDirective {
+  protected element: CdsAccordionPanel;
+
+  get cdsMotion() {
+    return this.element.cdsMotion;
+  }
+  @Input() set cdsMotion(value) {
+    this.element.cdsMotion = value;
+  }
+  get disabled() {
+    return this.element.disabled;
+  }
+  @Input() set disabled(value: boolean | '') {
+    this.element.disabled = !!value;
+  }
+  get expanded() {
+    return this.element.expanded;
+  }
+  @Input() set expanded(value: boolean | '') {
+    this.element.expanded = !!value;
+  }
+
+  @Output() cdsMotionChange: EventEmitter<CustomEvent> = new EventEmitter();
+
+  constructor(elementRef: ElementRef) {
+    this.element = elementRef.nativeElement;
+  }
+}
```

Comment:
Looking at the git ignore there were no changes to how the generated files are being ignored, so were these file unintentionally committed somehow?

Reply:
Yeah, these shouldn't be there. The !operator in the `.gitignore` doesn't seem to be working the way I'd expect it to.

I will go back and clean this up, this should not be committing the generated files to the repo. But in order to commit changes to the specs I have to comment out the `.gitignore` rule, make my changes and then re-enable the .gitignore rule. IMO - i do not think we should be committing generated files to the repo b/c the source and build configuration files (tsconfig.json) controls the generated output. And the generated output will be the same for each commit.

cc/ @bdryanovski - I think this covers your feedback to, correct?

Reply:
@hippee-lee - in that case - yes :) 

In some form or another, I think that having the generated files is a good idea - just because we could monitor the changes over time .. and have some indication that we are creating any change at all. In some cases, this could be even breaking change for example the case when we change a type or extend type in `cds/core` could have effect into `cds/angular` - right now we depend only on people to have this notion of linked changes at the back of there todo list.


But we could not implement this without a good way of committing and tracking changes without creating unwanted side-effects

Thanks, @coryrylan for noticing the `gitignore` file.

Reply:
All of the generated files and folders have been removed.